infra/build_specified_commit_test.py

```diff
@@ -66,39 +65,17 @@ def test_build_fuzzers_from_commit(self):
         old_commit,
         '/usr/local/google/home/lneat/Documents/oss-fuzz/infra/tmp',
         sanitizer='address')
-    old_error_code = self.reproduce_error(project_name, test_data, fuzzer)
+    old_error_code = reproduce_impl(project_name, fuzzer, False, [], [],
+                                    test_data)
     build_fuzzer_from_commit(
         project_name,
         new_commit,
         '/usr/local/google/home/lneat/Documents/oss-fuzz/infra/tmp',
```

Comment:
nit: take out the reference to your machine and replace it with something generic like /home/user